test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -20,934 +20,623 @@
+javax/net/ssl/DTLSv10/DTLSv10RehandshakeWithDataExTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/DTLSv10/DTLSv10SequenceNumberTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/DTLSv10/DTLSv10UnsupportedCiphersTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/FixingJavadocs/ImplicitHandshake.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/FixingJavadocs/KMTMGetNothing.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/FixingJavadocs/SSLSessionNulls.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/FixingJavadocs/SSLSocketInherit.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/HttpsURLConnection/CriticalSubjectAltName.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/HttpsURLConnection/Equals.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/HttpsURLConnection/GetResponseCode.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/AcceptLargeFragments.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/ArgCheck.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/Arrays.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/ExtendedKeyEngine.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/ExtendedKeySocket.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/LargeBufs.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-s390x,linux-x86,windows-all
+javax/net/ssl/SSLEngine/LargePacket.java https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/775 aix-all,linux-ppc64le,linux-x64,
```

Comment:
Please remove the trailing comma on this line.

Reply:
removed

Reply:
> removed

Instead of removing the comma, it looks like you added `windows-all`. That isn't consistent with your response. Please confirm that adding `windows-all` was the (original?) intent.

Reply:
right, forget to add windows-all, just added it